CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ We used to have incremental versioning before `0.1.0`.
 
 - Fixes that `_` was marked as invalid by `VagueImportViolation`
 - Fixes that docs for `VagueImportViolation` were misleading
+- Add more complex example to `ParametersIndentationViolation`
```

Comment:
Can you please also add #1020 to the list?

Reply:
Added. Also added issue numbers, is it ok?